app/Http/Controllers/API/GroupController.php

```diff
@@ -252,14 +261,25 @@ public static function getGroupList()
      */
 
     public static function listNamesv2(Request $request) {
+        $request->validate([
+            'archived' => ['string', 'in:true,false'],
```

Comment:
I'd probably call this parameter includeArchived, as when it's called 'archived', passing archived=true could make one think you would only be receiving archived.  Elsewhere we have 'includeDetails' so perhaps better for consistency, too.  That said, I can live with it if simpler to keep it.